src/UI/pages/source_list.tsx

```diff
@@ -5,22 +5,41 @@ import { AppDispatch } from '../state/store';
 import { useAppSelector } from '../state/hooks';
 import { useDispatch } from 'react-redux';
 import SingleSource from '../components/sources/singleSource';
+import InfiniteScroll from 'react-infinite-scroll-component';
 
 function SourcesView(): JSX.Element {
   const source_info = useAppSelector((state) => state.source.source_info);
-  const source_info_status = useAppSelector(
-    (state) => state.source.source_info_status
-  );
-
-  console.log(source_info);
 
   const dispatch = useDispatch<AppDispatch>();
   useEffect(() => {
     dispatch(getSourceInfo());
   }, [dispatch]);
 
+  const refresh = () => {};
+
   return (
     <>
+    <InfiniteScroll 
+        dataLength={source_info.length}
+        next={() => {
+          dispatch(getSourceInfo());
+        } }
+        hasMore={false}
```

Comment:
If `hasMore` is always set to false, it's never going to get more data... Is this intended?